BackEnd/profiles/serializers.py

```diff
@@ -378,3 +381,31 @@ def get_user_profile_name(self, obj) -> str:
 
     def get_company_name(self, obj) -> str:
         return obj.company_name
+
+
+class ProfileModerationSerializer(serializers.Serializer):
+    uid = serializers.CharField()
+    action = serializers.CharField()
+    timestamp = serializers.IntegerField()
+
+    def validate_uid(self, value):
+        try:
+            Profile.objects.get(pk=decode_uid(value))
```

Comment:
You already check this in View
get_object_or_404

Reply:
validate_uid removed